hazelcast/src/test/java/com/hazelcast/jet/core/ExecutionLifecycleTest.java

```diff
@@ -700,6 +703,166 @@ public void when_memberLeft_then_jobFails() {
                 .hasRootCauseInstanceOf(MemberLeftException.class);
     }
 
+    @Test
+    public void when_pmsInitBlocks_then_otherJobsNotBlocked() throws Exception {
+        // Given
+        DAG dagBlocking = new DAG().vertex(new Vertex("test",
+                new MockPMS(() -> new MockPS(MockP::new, MEMBER_COUNT)).initBlocks()));
+        DAG dagNormal = new DAG().vertex(new Vertex("test",
+                new MockPS(MockP::new, MEMBER_COUNT)));
+
+        List<Future<Job>> submitFutures = new ArrayList<>();
+
+        // When
+        int numJobs = 1;
```

Comment:
Could you increase the number of jobs to be more than both the operation thread count and coordination executor thread count since both of these thread pools may get stuck by these init/close calls. I'd suggest this change for all other tests below.

Reply:
I'm limited by the size of offloadable executor. If number of job exceeds the offloadable executor capacity, the tests are stuck even if functionality is ok.

Alternatively, I can "hack" to pass custom executor size 🤔

Reply:
Done